docker/ssl/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+nginx:
+  build: nginx-image
+  links:
+    - letsencrypt
+    - iri
+  environment:
+    - MY_DOMAIN_NAME=__DOMAIN__
+    - PORT=__PORT__
+  ports:
+    - "80:80"
+    - "443:443"
+  volumes_from:
+    - letsencrypt
+letsencrypt:
+  image: quay.io/letsencrypt/letsencrypt:latest
+  command:  bash -c "sleep 6 && certbot certonly --standalone --email "__EMAIL__" -d "__DOMAIN__" -n --agree-tos"
```

Comment:
Why the `sleep 6`? Are you trying to coordinate the `nginx` container startup with the `letsencrypt` one?